docs/building-spark.md

```diff
@@ -284,6 +284,7 @@ If use an individual repository or a repository on GitHub Enterprise, export bel
 
 ### Related environment variables
 
+<div class="table-wrapper">
```

Comment:
I'm not a CSS expert, but instead of adding a new `div` all over the place, couldn't we just add the appropriate styling in the main CSS file directly to all `table` elements?

Reply:
https://github.com/gettalong/kramdown/issues/69